README.md

```diff
@@ -5,11 +5,20 @@ JSON diffing on steroids.
 ## 🧬 Installation
 
 ```shell
-npm install joey-the-differ
+npm install mawrkus/joey-the-differ
 ```
 
 ## 🧬 Usage
 
+### Command-line
```

Comment:
Wdyt about naming this section "Docker"?

And if we place it just after the "Command line" usage, no need to add the "Checking help"